docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/configure-aws.rst

```diff
@@ -103,20 +103,20 @@ a secure 24 character password.
     the password from the password field into a secure location (e.g., a
     password manager).
 
-.. image:: ../../_assets/img/aws-wizard-step2.png
+.. image:: ../../../_assets/img/aws-wizard-step2.png
    :alt: CrateDB Cloud configuration wizard step 2
 
-You can also already set the scale unit of the cluster to the level you want
-here. As you move the slider horizontally, the different capacities
-corresponding to each scale unit within the subscription plan will be
-displayed. Currently, within each subscription plan clusters can be scaled
-between scale units 1-3. The default scale unit is 1. Note that scaling the
-cluster changes its price.
+Finally, you can also set the scale unit of the cluster to the desired level
+here. As you move the slider horizontally, you will move up (or down) the scale
+levels within the subscription plan you previously selected. As you will see,
+the hardware capacities of the cluster will change correspondingly. Currently,
+within each subscription plan clusters can be scaled between scale units 1-3.
+The default scale unit is 1. Note that scaling the cluster changes its price.
 
-Do not worry, however: clusters can be scaled up or down as needed, for example
-if your use case changes, at any point later on. To understand more about
-subscription plans and scale units, refer to our `reference on subscription
-plans`_.
+Do not worry, however: clusters can be scaled up or down as needed - for
```

Comment:
I guess a change of use case is an implicit reason why a user would scale their cluster and perhaps does not need to be exemplified.

Could this be re-written in a simpler way as follows:

```
Don't worry though, clusters can be scaled up or down as needed at any time  if your use case changes.
```

Reply:
No, because use case changes are only one reason to scale up or down; you can also have under/overestimated usage, or have financial reasons, etc.